site/documentation/content/admin-guide/monitoring-tracing-config.md

```diff
@@ -3,16 +3,32 @@ title = "Monitoring & Tracing"
+The individual components of an Eclipse Hono&trade; installation need to work together in order to provide their
+functionality to devices and applications. Under normal circumstances these interactions work flawlessly.
+However, due to the nature of distributed systems, any one (or more) of the components may crash or become otherwise
+unavailable due to arbitrary reasons. This page describes how Hono supports operations teams by providing insights
+into the individual service components and their interactions with each other by means of reporting metrics and
+tracing the processing of individual messages through the system.
 <!--more-->
 
-When a device uploads telemetry data to the HTTP adapter, the adapter invokes operations on the Device Registration, Credentials and the Tenant services in order to authenticate and authorize the device before sending the telemetry data downstream to the AMQP 1.0 Messaging Network. The overall success of this process and the latency involved before the message reaches the consumer is determined by the individual interactions between the service components.
+When a device uploads telemetry data to the HTTP adapter, the adapter invokes operations on the Device Registration,
+Credentials and the Tenant services in order to authenticate and authorize the device before sending the telemetry
+data downstream to the AMQP 1.0 Messaging Network. The overall success of this process and the latency involved before
+the message reaches the consumer is determined by the individual interactions between the service components.
 
 ## Monitoring
 
-In a production environment, an operations team will usually want to keep track of some *key performance indicators* (KPI) which allow the team to determine the overall *health* of the system, e.g. memory and CPU consumption etc. Hono supports the tracking of such KPIs by means of metrics it can be configured to report. The metrics are usually collected in a time series database like *InfluxDB* or *Prometheus* and then visualized on a monitoring dash-board built using frameworks like *Grafana*. Such a dash-board can usually also be configured to send alarms when certain thresholds are exceeded.
+In a production environment, an operations team will usually want to keep track of some *key performance indicators* (KPI)
+which allow the team to determine the overall *health* of the system, e.g. memory and CPU consumption etc.
+Hono supports the tracking of such KPIs by means of metrics it can be configured to report. The metrics are usually
+collected in a time series database like *InfluxDB* or *Prometheus* and then visualized on a monitoring dash-board built
```

Comment:
In the above paragraph, the reference to `InfluxDB` can be removed too.

Reply:
Influx DB is just used as another example here for time series DBs in general. There is no implication that Hono can directly write to InfluxDB.